docs/contributing/create-doc.md

```diff
@@ -73,15 +71,34 @@ As an example, let's say you needed to add a Best Practices doc to the [**Send D
 
 Once you decide on placement, use the card HTML code in that doc to create a new entry.
 
-## Step 7: Preview your changes
+## Step 7: Create CID URL
+
+Because docs links appear throughout our UI, and canonical links often change, we assign each doc a permalink URL containing a content ID (CID) number. The permalink URL does a 301 redirect to the canonical URL. This way, future changes to the canonical URL won't impact these UI links.
```

Comment:
```suggestion
Because docs links appear throughout our product, and canonical links often change, we assign each doc a permalink URL containing a content ID (CID) number. The permalink URL does a 301 redirect to the canonical URL. This way, future changes to the canonical URL (such as a product name change) won't impact the Learn More links for the user. And will save us code changes to the UI, providing great flexibility when we need to make quick changes.
```